app/controllers/file_controller.rb

```diff
@@ -50,7 +50,9 @@ def show
     disposition = params[:disposition] || 'inline'
     filename = params["filename"] || "#{objid}#{fileid}"
     headers['Content-Disposition'] = "#{disposition}; filename=#{filename}"
-    if ds.mimeType
+    if filename.include?('.jpg')
```

Comment:
@hweng - Option 2 please. The current PR fixes the symptom (Calisphere getting files with the wrong content type header), but not the problem. The underlying problem is the mimetype is not available for those images, so that first `if` condition check that we expect to work, doesn't. Fixing the derivatives/technical metadata should in turn fix the symptom, the content type header being incorrect. We've done this kind of thing before, but in more targeted use cases. The hope here is to do something more broad.

Calisphere does harvest a lot of our metadata. I'm not sure whether they do anything explicitly with our technical metadata. But Calisphere aren't only people who are downloading our files, so we need a more global solution that addresses the underlying problem. I apologize if this could have been better clarified in the original ticket. Perhaps it was too focused on the specific example.